draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -249,6 +249,8 @@ information.
 * `vct`
     * REQUIRED. The type of the Verifiable Credential, e.g.,
 `https://credentials.example.com/identity_credential`, as defined in (#type-claim).
+* `vct#integrity`
+    * OPTIONAL hash of the Type Metadata document to guarantee integrity as defined in Section (#document-integrity).
```

Comment:
```suggestion
    * OPTIONAL. The hash of the Type Metadata document to guarantee integrity as defined in Section (#document-integrity).
```

I apologize for being so picky.

I also noticed that the spec uses _Type Metadata document_ and _Type Metadata Document_ at the same time.

Reply:
> I also noticed that the spec uses _Type Metadata document_ and _Type Metadata Document_ at the same time.

Looks like there's one use of "Document" - maybe do a PR to make that big D a little one?

Reply:
PR https://github.com/oauth-wg/oauth-sd-jwt-vc/pull/341 is open

Reply:
I'd kinda meant that as a suggestion for @adeinega but thank you @cre8!

Reply:
Good point @adeinega and note that the "as defined in Section (#somewhere)" results in the word Section showing up twice.  
<img width="547" height="181" alt="Screenshot 2025-09-02 at 12 26 18 PM" src="https://github.com/user-attachments/assets/8699ae26-baa7-4d00-8623-c3d5c78b1981" />